ZenPacks/zenoss/WSMAN/datasources/WSMANDataSource.py

```diff
@@ -276,13 +275,14 @@ def onError(self, result, config):
         log.error('%s %s', config.id, errmsg)
 
         data = self.new_data()
+        eventClass = config.datasources[0].eventClass
         data['events'].append({
             'eventClassKey': 'wsmanCollectionError',
-            'eventClass': '/Status/Events',
+            'eventClass': eventClass or '/Status/Events',
```

Comment:
You would want to set the default eventClass on WSMAN datasource level, an example -https://github.com/zenoss/ZenPacks.zenoss.CalculatedPerformance/blob/develop/ZenPacks/zenoss/CalculatedPerformance/datasources/AggregatingDataSource.py#L37